fairscale/experimental/wgit/cli.py

```diff
@@ -71,25 +72,30 @@ def main(argv=None):
     args = parser.parse_args(argv)
 
     if args.command == "init":
-        weigit = weigit_api.WeiGit()
+        repo = Repo(Path.cwd(), init=True)
 
     if args.command == "add":
-        weigit_api.WeiGit.add(args.add)
+        repo = Repo(Path.cwd())
+        repo.add(args.add)
 
     if args.command == "status":
-        weigit_api.WeiGit.status()
+        repo = Repo(Path.cwd())
+        repo.status()
 
     if args.command == "log":
-        weigit_api.WeiGit.log(args.file)
+        repo = Repo(Path.cwd())
+        repo.log(args.file)
 
     if args.command == "commit":
-        weigit_api.WeiGit.commit(args.message)
+        repo = Repo(Path.cwd())
+        repo.commit(args.message)
 
     if args.command == "checkout":
-        weigit_api.WeiGit.checkout()
+        repo = Repo(Path.cwd())
+        repo.checkout(args.checkout)
 
     if args.command == "version":
-        print(wgit.__version__)
+        print(".".join([str(x) for x in version.__version_tuple__]))
```

Comment:
it seems that you can just use __version__ directly not duplicate the code here?